recipe/meta.yaml

```diff
@@ -1,18 +1,18 @@
 {% set name = "sentineleof" %}
-{% set version = "0.4.2" %}
+{% set version = "0.4.3" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 7761556af99693e70abe56f2743be88d1c40e04fd7944efac4e41851ccf33c4a
+  sha256: 24186c7b9690abad6382388adc90a2512b219b852f40ab164641ee9122441ace
 
 build:
   noarch: python
```

Comment:
Despite of the linter recommendation, putting a minimum version in noarch seems not to work.
IMHO it is OK for now to leave it as is.

Reply:
alright good to know